classes/webservice/WebserviceSpecificManagementImages.php

```diff
@@ -62,11 +62,6 @@ class WebserviceSpecificManagementImagesCore implements WebserviceSpecificManage
      */
     protected $imageType = null;
 
-    /**
-     * @var int The maximum size supported when uploading images, in bytes
-     */
-    protected $imgMaxUploadSize = 3000000;
```

Comment:
hmm also this is probably a BC break. I think its acceptable, but should be defined in PR description